### DISCUSSION TITLE
[A/L] (2021/22) Relazioni - Esercizio 16

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#288] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152319682-143fcd77-b271-4d41-8cc3-6635c14f8be5.png)

**Comment by LuginiAndrea** [comment]
Stessa soluzione dell'esercizio 15 (#287 )
